### PR TITLE
feat: Initial implementation

### PR DESCRIPTION
**Pull request by stipsan**
Based on https://github.com/eik-lib/rollup-plugin-import-map

Merging this will trigger a release using semantic release.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macOS-latest, windows-latest]
+        node-version: [10.x, 12.x, 13.x]
```
Comment (trygve-lie) [inline .github/workflows/test.yml:11]: Remove 10 and 13. Add 14.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@eik/postcss-import-map",
+  "version": "1.0.0",
+  "description": "PostCSS plugin that uses Eik defined import map files to transform bare import specifiers to absolute URLs in @import rules",
+  "main": "dist/plugin.cjs.js",
+  "files": [
+    "src",
+    "dist"
+  ],
```
Comment (trygve-lie) [inline package.json:9]: Add README, LISENCE and CHANGELOG.

Comment (trygve-lie) [reply package.json:9]: And package.json

File: renovate.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      "automerge": true,
+      "major": {
+        "automerge": false
+      }
+    },
+    {
+      "depTypeList": ["devDependencies"],
+      "schedule": ["before 4am on monday"],
+      "commitMessage": "Dev dependency {{depName}} updated to version {{newValue}}",
+      "prTitle": "Dev dependency update: {{depName}}",
```
Comment (trygve-lie) [inline renovate.json:15]: `commitMessage` and `prTitle` can be removed.

**Comment by trygve-lie** [comment]
Looks good. Do you think we could avoid the `.mjs` extension?

File: package.json (hunk 2 above)
Comment (stipsan) [reply package.json:9]: README, LISENCE and package.json are always included by npm. Verified by running npm pack locally. I'll add CHANGELOG 🙂

**Comment by stipsan** [comment]
> Looks good. Do you think we could avoid the `.mjs` extension?

Yup, removed 👍

**Pull request merged**